src/main/java/com/simprints/libsimprints/Identification.java

```diff
@@ -23,14 +24,16 @@ public Identification() {
      * @param guid       A string containing the guid
      * @param confidence A float containing the (matching) confidence
      */
-    public Identification(String guid, float confidence) {
+    public Identification(String guid, float confidence, Tier tier) {
```

Comment:
This change breaks the [Identification unit tests](https://github.com/SimPrints/LibSimprints/blob/master/src/androidTest/java/com/simprints/libsimprints/IdentificationTest.java#L19). Can you update them?